packages/nexus-sdk/src/Resource/types.ts

```diff
@@ -1,5 +1,17 @@
 import { Context, Resource, PaginatedResource } from '../types';
 
+export type ExternalLink = {
+  '@id': string;
+  '@type'?: Resource['@type'];
+  paths: string[];
+};
+
+export type InternalLink = Resource & {
```

Comment:
Why are `InternalLink` and `ExternalLink` different? They look the same according to the doc.

Also, why is `InternalLink` a intersection of `paths` and `Resource` but `External` is not?